src/components/layout/Quiz/index.js

```diff
@@ -0,0 +1,77 @@
+import React from "react"
+import { Redirect } from "react-router-dom"
+import { connect } from "react-redux"
+import PropTypes from "prop-types"
+
+import Compass from "../../../images/compass.png"
+
+// Redux
+import { setLocation } from "../../../actions/quizActions"
+
+// Material UI
+import {
+  Box,
+  Button,
+  CardMedia,
+  Container,
+  Typography,
+} from "@material-ui/core"
+
+// Components
+import StateQuestion from "./StateQuestion"
+
+const Quiz = ({ quiz, setLocation }) => {
+  if (!quiz.agreement) {
+    return <Redirect to="/" />
+  }
+
+  return (
+    <Container maxWidth="lg">
+      <Box display={"flex"} alignItems={"center"} mb={2}>
+        <CardMedia src={Compass}>
+          <img src={Compass} />
+        </CardMedia>
+
+        <Box ml={2}>
+          <Typography variant={"h1"}>Compensation Compass</Typography>
```

Comment:
That's a good callout. The design has the title on this page left-aligned, so I think it would make sense to move the Terms and Conditions title, and, like you said, create a page title component that does the styling for us.